dhcpv4/types.go

```diff
@@ -270,7 +293,12 @@ const (
 	OptionSIPUAConfigurationServiceDomains      optionCode = 141
 	OptionOPTIONIPv4AddressANDSF                optionCode = 142
 	OptionOPTIONIPv6AddressANDSF                optionCode = 143
-	// Options 144-149 returned in RFC 3679
+	OptionGeoLoc                                optionCode = 144
+	OptionFORCERENEWNONCECAPABLE                optionCode = 145
+	OptionRDNSSSelection                        optionCode = 146
+	OptionOPTIONV4DOTSRI                        optionCode = 147
+	OptionOPTIONv4DOTSADDRESS                   optionCode = 148
```

Comment:
Any reason why address is all upper case instead of having only the 1st letter as uppercase?